mackerel-plugin-mongodb/lib/mongodb.go

```diff
@@ -183,12 +184,18 @@ type MongoDBPlugin struct {
 }
 
 func (m MongoDBPlugin) fetchStatus() (bson.M, error) {
-	session, err := mgo.Dial(m.URL)
+	mongoDBDialInfo := &mgo.DialInfo{
+		Addrs:    []string{m.URL},
```

Comment:
I could not establish an authenticated connection (with `-username` and `-password`) .
```
$ go run main.go -username bob -password 1234
2017/12/27 18:19:16 OutputValues:  no reachable servers
exit status 1
```

Can you confirm it?
(If the change will be big, we can go back to the first commit https://github.com/mackerelio/mackerel-agent-plugins/pull/467/commits/be0b799079122699335a89bb26dea2a818122926 and simply change the mode to `Eventual`. )

Reply:
@astj 
I have been updated with parameter options (-username and -password).
Could you please confirm it?
Thanks you!

Reply:
Confirmed working. :+1: